src/async/stompest/async/client.py

```diff
@@ -135,14 +135,16 @@ def session(self):
     # STOMP commands
     #
     @defer.inlineCallbacks
-    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None):
-        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None)
+    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None):
+        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None)
```

Comment:
Instead of passing these all the way through, could they instead be obtained from the SSLContext object, assuming you used `.load_cert_chain()` like you do for the sync case?